distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+=== Metacards
+
+A metacard is a single instance of metadata in the Catalog (an instance of a metacard type) which
+generally contains general information about the product, such as the title of the product, the product's geo-location, the date the product was created and/or modified, the owner or producer, and/or the security classification. 
+
+==== Metacard Type
+
+A metacard type indicates the attributes available for a particular metacard.
+It is a model used to define the attributes of a metacard, much like a schema.
+
+A metacard type indicates the attributes available for a particular type of data.
+For example, an image may have different attributes than a PDF document, so each could be defined to have their own metacard type.
+
+===== Default Metacard Type and Attributes
+
+Most metacards within the system are created using the default metacard type or a metacard type based on the default type.
+The default metacard type of the system can be programmatically retrieved by calling `ddf.catalog.data.BasicTypes.BASIC_METACARD`.
+The name of the default `MetacardType` can be retrieved from `ddf.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+
+The default metacard type has the following required attributes.
+Though the following attributes are required on all metacard types, setting their values is optional except for ID.
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
+[cols="4*", options="header"]
+|===
+
+|`Metacard` Constant
+|Attribute Name
+|Attribute Format
+|Description
+
+|`CONTENT_TYPE`
+|`metadata-content-type`
+|STRING
+|Attribute name accessing for the metadata content type of a Metacard.
+
+|`CONTENT_TYPE_VERSION`
+|`metadata-content-type-version`
+|STRING
+|Attribute name for the version of the metadata content accessing type of a Metacard.
+
+|`CREATED`
+|`created`
+|DATE
+|Attribute name for the date/time *this Metacard* was created.
+
+|`EFFECTIVE`
+|`effective`
+|DATE
+|Attribute name for the date/time of *the product* represented by the Metacard.
+
+|`EXPIRATION`
+|`expiration`
+|DATE
+|Attribute name for the date/time the Metacard is no longer valid and can be removed.
+
+|`GEOGRAPHY`
+|`location`
+|GEOMETRY
+|Attribute name for the location for this Metacard.
+
+|`ID`
+|`id`
+|STRING
+|Attribute name for the ID of the Metacard.
+
+|`METADATA`
+|`metadata`
+|XML
+|Attribute name for the XML metadata for this Metacard.
+
+|`MODIFIED`
+|`modified`
+|DATE
+|Attribute name for the date/time this Metacard was last modified.
+
+|`RESOURCE_SIZE`
+|`resource-size`
+|STRING
+|Attribute name for the size in bytes of the product this Metacard represents.
+
+|`RESOURCE_URI`
+|`resource-uri`
+|STRING
+|Attribute name for the URI reference to the product this Metacard represents.
+
+|`TARGET_NAMESPACE`
+|`metadata-target-namespace`
+|STRING
+|Attribute name for the target namespace of the metadata content type of a Metacard.
```

Comment:
Documentation indicates this is intended to be deprecated, although the code does not reflect this. https://codice.atlassian.net/wiki/display/DDF/%5Bdeprecated%5D+metadata-target-namespace